lotus/sem_ops/cascade_utils.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+
+import lotus
+
+
+def importance_sampling(
+    proxy_scores: list[float],
+    sample_percentage: float,
+):
```

Comment:
Would be helpful to have output types here

Reply:
Added for all util funcs in the file!